docs/recorder.md

```diff
@@ -0,0 +1,15 @@
+The recorder works by recording incoming, outgoing messages, the times at which nodes connected/disconnected, times at which itself stopped and started. 
```

Comment:
Please use bullet points for more readable doc.

Reply:
Added